koku/masu/util/aws/common.py

```diff
@@ -871,6 +870,8 @@ def clear_s3_files(
                     exc_info=err,
                 )
     delete_s3_objects(request_id, to_delete, context)
+    manifest = ReportManifestDBAccessor().get_manifest_by_id(manifest_id)
+    ReportManifestDBAccessor().mark_s3_parquet_cleared(manifest)
```

Comment:
Do we need to create two `ReportManifestDBAccessor` instances? These methods look like they could be class methods. Otherwise, create an instance once instead of avoiding the overhead of creating the instance twice.

Reply:
Yeh that was a mistake for sure. Thanks.